app/controllers/concerns/idv_step_concern.rb

```diff
@@ -20,7 +20,14 @@ def step_attempts_exceeded?
   end
 
   def confirm_step_allowed
-    redirect_to_fail_url if step_attempts_exceeded?
+    if step_attempts_exceeded?
```

Comment:
Not sure it's in the AC but how about logic to only log it once when the limit is reached?  (vs on every retry)

Reply:
On second thought we can just query distinct.  Probably better to log it.

Reply:
Flip flopping again.  Def better to know if they hit the limit separately (and only once).  Hitting the limit, waiting the window and getting throttled again.  Should be separate from 2 tries when throttled once.

Reply:
We don't have a good way to only log once unfortunately. We initially had the event fire when the throttler incremented to be on the final attempt, but since it's possible for the final attempt to succeed that wasn't a great way to capture that someone was throttled. We typically only have throttler checks at the beginning of an action, which means it can happen multiple times :/

For reference, this builds on work from #4829

Reply:
We could have two separate events or one event and a boolean which indicates they are retrying and still throttled

Reply:
"We don't have a good way to only log once unfortunately." - We'd have to keep incrementing but that's a big change.  I'll go ahead and approve